### PR TITLE
chore: update volumes in lib-gen nightly CI

### PR DESCRIPTION
**Pull request by diegomarquezp**
Test in progress: https://github.com/googleapis/google-cloud-java/actions/runs/8159602869/job/22304275554

### CODE REVIEW

**Comment by diegomarquezp** [comment]
image

File: .github/workflows/verify-generation-config.yaml [hunk 1]
```diff
@@ -17,15 +17,16 @@ jobs:
         if [[ $(docker volume inspect repo) != '[]' ]]; then
           docker volume rm repo
         fi
-        docker volume create --name "repo" --opt "type=none" --opt "device=$(pwd)" --opt "o=bind"
+        docker volume create --name "repo-google-cloud-java" --opt "type=none" --opt "device=$(pwd)" --opt "o=bind"
```
Comment (burkedavison) [inline .github/workflows/verify-generation-config.yaml:20]: With this new name, should the above logic be updated?

Comment (diegomarquezp) [reply .github/workflows/verify-generation-config.yaml]: Ah, thanks for the catch. It indeed needs to be updated. Done.

**Review by JoeWang1127 — APPROVED** [review]
Thanks for the PR.

Could you create an issue and root cause for this PR?

**Comment by diegomarquezp** [comment]
> Thanks for the PR.
> 
> Could you create an issue and root cause for this PR?

Yes, I created https://github.com/googleapis/sdk-platform-java/issues/2536 . Thank you.

**Pull request merged**